### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by donbeave** [rsocket/rsocket-rpc-java#71]
Hello!

RSocket RPC looks very promising for me but makes me surprise this project doesn't have any release since Aug 2019. Is it abandoned?

**Comment by OlegDokuka** [comment]
Hey! @donbeave! I'm actively working on the last bits for @rsocket-java. Once it is in good shape, I will be cleaning this project up to make it more attractive for everyone (more docs, samples, etc).

Stay tuned!

**Comment by donbeave** [comment]
@OlegDokuka awesome! Thank you!

**Comment by xmlking** [comment]
@OlegDokuka any update on this project? 
 interested to migrate my **gRPC** µServices to **rSocket** and felt this project might be ideal path.  But worried on maintenance  of this project

**Comment by iamceph** [comment]
> @OlegDokuka any update on this project? interested to migrate my **gRPC** µServices to **rSocket** and felt this project might be ideal path. But worried on maintenance of this project

Exactly, I find RSocket really awesome, it seems that there is just not enough support :/

**Comment by ummels** [comment]
@OlegDokuka Any update on the status of this project?
